File: sample-app-java+push-notification/src/main/java/com/cometchat/sampleapp/java/fcm/utils/AppConstants.java
package com.cometchat.sampleapp.java.fcm.utils;

public class AppConstants {

    public static Boolean isQRScanned = false;
    /**
     * Error messages used throughout the application.
     */
    public static class ErrorConstants {
        public static final String USER_NOT_FOUND = "ERR_UID_NOT_FOUND";
    }

    /**
     * Success messages or constants used throughout the application.
     */
    public static class SuccessConstants {
        public static final String SUCCESS = "success";
        public static final String ALREADY_MEMBER = "Member already has the same scope participant";
    }

    /**
     * Constants used for the app.
     */
    public static class JSONConstants {
        public static final String SAMPLE_APP_USERS_URL = "https://assets.cometchat.io/sampleapp/sampledata.json";
        public static final String KEY_USER = "users";
        public static final String UID = "uid";
        public static final String NAME = "name";
        public static final String AVATAR = "avatar";
        public static final String RAW_JSON = "rawJson";
        public static final String REPLY_COUNT = "reply_count";
    }

    /**
     * Constants used for FCM.
     */
    public static class FCMConstants {
        public static final String PROVIDER_ID = "Android-CometChat-Team-Messenger";
        public static final String KEY_UID = "UID";
        public static final String KEY_DATA = "FCM_DATA";
        public static final String REPLY_FROM_NOTIFICATION = "key_text_reply";
        public static final String KEY_NOTIFICATION_ID = "FCM_NOTIFICATION_ID";
        public static final String KEY_CLICKED_NOTIFICATION_ID = "FCM_CLICKED_NOTIFICATION_ID";
        public static final String DEFAULT_NOTIFICATION_CHANNEL_ID = "CometChatTeamMessenger"; // This is the ID for the default notification channel.
        public static final String CALL_NOTIFICATION_CHANNEL_ID = "Call"; // This is the message-specific notification channel ID.
        public static final String MESSAGE_NOTIFICATION_CHANNEL_ID = "Message"; // This is the message-specific notification channel ID.
        public static final String MESSAGE_NOTIFICATION_CHANNEL_NAME = "Message notification"; // This is the name of the message notification channel.
        public static final String GROUP_KEY = "CometChatNotificationGroup"; // This is the name of the message notification channel.
        public static final int NOTIFICATION_GROUP_SUMMARY_ID = 2000; // This ID is used for notification grouping.
        public static final String KEY_NOTIFICATION_SUMMARY_ID = "FCM_NOTIFICATION_SUMMARY_ID";
        public static final String NOTIFICATION_TYPE = "NOTIFICATION_TYPE";
        public static final String NOTIFICATION_TYPE_MESSAGE = "NOTIFICATION_TYPE_MESSAGE";
        public static final String NOTIFICATION_TYPE_CALL = "NOTIFICATION_TYPE_CALL";
        public static final String NOTIFICATION_PAYLOAD = "NOTIFICATION_DATA";
        public static final String NOTIFICATION_REPLY_ACTION = "NOTIFICATION_REPLY_ACTION";
        public static final String NOTIFICATION_KEY_REPLY_ACTION = "Reply";
    }
}

File: sample-app-kotlin+push-notification/src/main/java/com/cometchat/sampleapp/kotlin/fcm/utils/AppConstants.kt
package com.cometchat.sampleapp.kotlin.fcm.utils

class AppConstants {

    companion object {
        var IS_QR_SCANNED: Boolean = false
    }

    /** Error messages used throughout the application.  */
    object ErrorConstants {
        const val USER_NOT_FOUND: String = "ERR_UID_NOT_FOUND"
    }

    /** Success messages or constants used throughout the application.  */
    object SuccessConstants {
        const val SUCCESS: String = "success"
        const val SUCCESS_ALREADY_MEMBER: String = "Member already has the same scope participant"
    }

    object JSONConstants {
        const val SAMPLE_APP_USERS_URL: String = "https://assets.cometchat.io/sampleapp/sampledata.json"
        const val KEY_USER: String = "users"
        const val UID: String = "uid"
        const val NAME: String = "name"
        const val AVATAR: String = "avatar"
        const val RAW_JSON: String = "raw_json"
        const val REPLY_COUNT: String = "reply_count"
    }

    object FCMConstants {
        const val PROVIDER_ID: String = "Android-CometChat-Team-Messenger"
        const val KEY_UID: String = "UID"
        const val KEY_DATA: String = "FCM_DATA"
        const val REPLY_FROM_NOTIFICATION: String = "key_text_reply"
        const val KEY_NOTIFICATION_ID: String = "FCM_NOTIFICATION_ID"
        const val KEY_CLICKED_NOTIFICATION_ID: String = "FCM_CLICKED_NOTIFICATION_ID"
        const val DEFAULT_NOTIFICATION_CHANNEL_ID: String = "CometChatTeamMessenger" // This is the ID for the default notification channel.
        const val MESSAGE_NOTIFICATION_CHANNEL_ID: String = "Message" // This is the message-specific notification channel ID.
        const val MESSAGE_NOTIFICATION_CHANNEL_NAME: String = "Message notification" // This is the name of the message notification channel.
        const val GROUP_KEY: String = "CometChatNotificationGroup" // This is the name of the message notification channel.
        const val NOTIFICATION_GROUP_SUMMARY_ID: Int = 2000 // This ID is used for notification grouping.
        const val KEY_NOTIFICATION_SUMMARY_ID: String = "FCM_NOTIFICATION_SUMMARY_ID"
        const val NOTIFICATION_TYPE: String = "NOTIFICATION_TYPE"
        const val NOTIFICATION_TYPE_MESSAGE: String = "NOTIFICATION_TYPE_MESSAGE"
        const val NOTIFICATION_PAYLOAD: String = "NOTIFICATION_DATA"
        const val NOTIFICATION_REPLY_ACTION: String = "NOTIFICATION_REPLY_ACTION"
        const val NOTIFICATION_KEY_REPLY_ACTION: String = "Reply"
    }
}
